docs/api-docs/partner/Dev-portal-overview.md

```diff
@@ -0,0 +1,98 @@
+Let's first examine how you can discover, display, and manage apps on BigCommerce stores.
+
+### Discovery
+Approved apps are listed on the app Marketplace for merchants to browse, search, and install.
+![App Marketplace](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-01-introduction-01.png "App Marketplace")
+
+### Management
+
+You can manage apps installed on a store in the store's control panel under **Apps** > [**My Apps**](https://login.bigcommerce.com/deep-links/manage/marketplace/apps/my-apps/drafts). Here, store users can launch or uninstall the app.
+
+![App Management](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-01-introduction-02.png  "App Management")
+
+### Display
+When a merchant launches an app, BigCommerce makes a request to the app's `/load` callback and displays the app's UI in an iFrame.
+
+![App Display](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-01-introduction-03.png  "App Display")
+
+## Use cases
+What can developers do with BigCommerce apps?
+
+### Integrate with Business Systems
+Use [BigCommerce APIs](https://developer.bigcommerce.com/api-reference) to integrate stores with business systems such as the following:
+* [ERP systems](https://www.bigcommerce.com/apps/erp/)
+* [Order management platforms](https://www.bigcommerce.com/apps/catalog-order-management/)
+* [Accounting software](https://www.bigcommerce.com/apps/accounting-tax/)
+* [Shipping and Fulfillment services](https://www.bigcommerce.com/apps/shipping-fulfillment/)
+
+[Learn more about BigCommerce's APIs](https://developer.bigcommerce.com/api-reference).
+
+
+### Connect to Sales Channels
+List products on external [sales channels](https://www.bigcommerce.com/apps/sales-channels/) with [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/channels-toolkit-reference).
+
+[Learn more about connecting apps to sales channels.](https://developer.bigcommerce.com/api-docs/channels/overview).
+
+### Add and Manage Scripts
+Create and manage scripts with [Scripts API](https://developer.bigcommerce.com/api-reference/store-management/scripts) and [Script Manager](https://support.bigcommerce.com/s/article/Using-Script-Manager).
+
+[Learn more about creating scripts](https://developer.bigcommerce.com/api-docs/storefront/scripts-overview).
+
+### Create Custom Widgets
+Create [widgets](https://support.bigcommerce.com/s/article/Page-Builder#builder) for merchants to install and configure.
+
+[Learn more about creating widgets](https://developer.bigcommerce.com/api-docs/storefront/widgets/widgets-overview).
+
+## Developing your app
+
+Here's how to develop your app with BigCommerce:
+
+1. [Start a free trial](https://www.bigcommerce.com/essentials/free-trial/) to test your apps.
+
+2. [Apply to be a Partner](https://www.bigcommerce.com/partners) to get paid for your apps.
+
```

Comment:
@traci707 Can you remove the new lines between each step? It causes extra spacing.

Reply:
removed lines